website/docs/runtime.md

```diff
@@ -101,22 +100,150 @@ etc.) or a completely untyped language like Ruby; a
 runtime checks. But it's precisely these runtime checks that make it easier to
 drive adoption of types in the long run.
 
-## Disabling runtime checks
+## Changing the runtime behavior
```

Comment:
Before going in depth with how to change the runtime behaviour, it may be better to give an example as why you would want to change its behaviour.

I think it also worth noting that code raising no error with Sorbet static could actually raise some with Sorbet runtime (and even in production).

Reply:
Good suggestion, thanks!

Reply:
Added blurb + snippet above.